src/main/resources/connectfour/end-view.css

```diff
@@ -0,0 +1,43 @@
+@font-face {
+    font-family: 'Nexa Light';
+    src: url("fonts/Nexa-Light.otf");
+}
+
+@font-face {
+    font-family: 'Nexa Bold';
+    src: url("fonts/Nexa-Bold.otf");
+}
+
+.gridpane-background {
+    -fx-background-color: #125B50;
+}
+
+.gridpane-text-container {
+    -fx-background-radius: 10px;
+    -fx-padding: 30px;
+}
+
+.text-player-information {
+    -fx-font-family: "Nexa Bold";
+    -fx-fill: #FFFFFF;
+    -fx-font-size: 46 px;
+}
+
+.text-score-information {
+    -fx-font-family: "Nexa Bold";
+    -fx-fill: #FFFFFF;
+    -fx-font-size: 40 px;
+}
+
+.button-play-and-exit {
```

Comment:
I would reccomend to add styles more like buttons :) For example:
![image](https://user-images.githubusercontent.com/65082704/171869394-f6e3ff56-ca35-4126-b933-e72346d7955c.png)